### DISCUSSION TITLE
exclude some Active Directory  groups from search

### DISCUSSION THREAD

**Discussion by turovskysd** [Yvand/LDAPCP#216] [Q&A] [answered]
Hello Yvand.
Is it possible in LDAPCPSE add filter to exclude some Active Directory groups (for example "c:0-.t|adfssso|domain\gsharepoint1" and "c:0-.t|adfssso|domain\gsharepoint2") from search in people picker when we try add group to the site collection?
If yes, could you help with parameters how realize it?

Thank you!
![LDAPCPSE-ADGroups](https://github.com/Yvand/LDAPCP/assets/20659882/d9faca53-cbe7-4b35-baf9-dc8c5524c4d6)

**Comment by Yvand** [comment, marked as answer]
In the role claim type entry, you can set the `Additional LDAP filter` as below:

To exclude 1 group:
`(!(sAMAccountName=group1))`

To exclude 2 groups:
`(!(sAMAccountName=group1)) (!(sAMAccountName=group2))`

**Comment by turovskysd** [comment]
Hello Yvand.
I had found solution

(!(&(sAMAccountName=GSTest))) (!(&(sAMAccountName=GSTest2)))

Thank you for your answer!
